src/main/java/io/confluent/connect/jdbc/sink/PreparedStatementBinder.java

```diff
@@ -93,20 +110,21 @@ protected int bindKeyFields(SinkRecord record, int index) throws SQLException {
 
       case KAFKA: {
         assert fieldsMetadata.keyFieldNames.size() == 3;
-        bindField(index++, Schema.STRING_SCHEMA, record.topic());
-        bindField(index++, Schema.INT32_SCHEMA, record.kafkaPartition());
-        bindField(index++, Schema.INT64_SCHEMA, record.kafkaOffset());
+        bindField(index++, Schema.STRING_SCHEMA, record.topic(), null, "");
+        bindField(index++, Schema.INT32_SCHEMA, record.kafkaPartition(), null, "");
+        bindField(index++, Schema.INT64_SCHEMA, record.kafkaOffset(), null, "");
```

Comment:
Can you actually fill in the real parameters here? The names of these columns are in `JdbcSinkConfig.DEFAULT_KAFKA_PK_NAMES`, and we should use the real table definition if it's available.

Reply:
> Can you actually fill in the real parameters here? The names of these columns are in `JdbcSinkConfig.DEFAULT_KAFKA_PK_NAMES`, and we should use the real table definition if it's available.

Since the table definition is null, the last parameter will just get ignored.